helm-charts/yunikorn/README.md

```diff
@@ -59,33 +59,49 @@ helm install yunikorn yunikorn/yunikorn
-| `service.portWeb`                      | Port of the web application service                    | `9889`
-| `resources.requests.cpu`               | CPU resource requests                                  | `200m`
-| `resources.requests.memory`            | Memory resource requests                               | `1Gi`
-| `resources.limits.cpu`                 | CPU resource limit                                     | `4`
-| `resources.limits.memory`              | Memory resource limit                                  | `2Gi`
-| `embedAdmissionController`             | Flag for enabling/disabling the admission controller   | `true`
-| `enableSchedulerPlugin`                | Flag for enabling/disabling scheduler plugin mode      | `false`
-| `operatorPlugins`                      | Scheduler operator plugins                             | `general`
-| `nodeSelector`                         | Scheduler deployment nodeSelector(s)                   | ` `
+| Parameter                                       | Description                                          | Default                         |
+| ----------------------------------------------- | ---------------------------------------------------- | ------------------------------- |
+| `imagePullSecrets`                              | Docker repository secrets                            | ` `                             |
+| `serviceAccount`                                | Service account name                                 | `yunikorn-admin`                |
+| `replicaCount`                                  | Scheduler replicas to be deployed                    | `1`                             |
+| `image.repository`                              | Scheduler image repository                           | `apache/yunikorn`               |
+| `image.tag`                                     | Scheduler image tag                                  | `scheduler-latest`              |
+| `image.pullPolicy`                              | Scheduler image pull policy                          | `Always`                        |
+| `pluginImage.repository`                        | Scheduler plugin image repository                    | `apache/yunikorn`               |
+| `pluginImage.tag`                               | Scheduler plugin image tag                           | `scheduler-plugin-latest`       |
+| `pluginImage.pullPolicy`                        | Scheduler plugin image pull policy                   | `Always`                        |
+| `webImage.repository`                           | Web app image repository                             | `apache/yunikorn`               |
+| `webImage.tag`                                  | Web app image tag                                    | `web-latest`                    |
+| `webImage.pullPolicy`                           | Web app image pull policy                            | `Always`                        |
```

Comment:
Now that we are updating and changing we should use a consistent naming convention here and make this `web.image.pullPolicy` etc Inline with the `admissionController`